github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/events/impl/commitchecks/GitHubBranchCommitCheck.java

```diff
@@ -30,4 +31,16 @@ public GitHubBranchCommitCheckDescriptor getDescriptor() {
      * @return <code>GitHubBranchCause</code> instance indicating if the build should be skipped, <code>null</code> otherwise.
```

Comment:
nothing changed in terms of how this part works, it only returns a `cause` object if the branch should be skipped. in the event the `local` object is null and there is no previous commit to generate a comparison range against, it just grabs the last commit and checks to see if that message indicates a skip.